client-app/pages/account/loyalty.vue

```diff
@@ -0,0 +1,200 @@
+                    </td>
+                  </tr>
+                </template>
+                <template #desktop-body>
+                  <tr
+                    v-for="operation in userBalance?.pointsOperations"
+                    :key="operation.id"
+                    class="even:bg-gray-50 hover:bg-gray-200 cursor-pointer"
+                  >
+                    <td class="p-5 overflow-hidden overflow-ellipsis">
+                      {{ moment(operation.createdDate).format("YYYY-MM-DD") }}
+                    </td>
+                    <td class="p-5 overflow-hidden overflow-ellipsis">
+                      {{ operation.reason }}
+                    </td>
+                    <td class="p-5 overflow-hidden overflow-ellipsis">{{ operation.isDeposit }}</td>
+                    <td class="p-5 overflow-hidden overflow-ellipsis">{{ operation.amount }}</td>
+                    <td class="p-5 overflow-hidden overflow-ellipsis">
+                      {{ operation.balance }}
+                    </td>
+                  </tr>
+                </template>
+                <template #desktop-empty>
+                  <!-- Workaround for using colspan -->
+                  <tr>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                  </tr>
+                  <tr>
+                    <td colspan="5" class="polygons-bg">
+                      <div class="flex items-center pl-56 space-x-10 h-80">
+                        <img src="/static/images/account/icons/no-addresses.svg" alt="No operations" />
+                        <div class="flex flex-col space-y-2">
+                          <span class="text-base">There are no operation yet</span>
+                        </div>
+                      </div>
+                    </td>
+                  </tr>
+                </template>
+              </VcTable>
+              <VcInfinityScrollLoader
+                v-if="isInfinityScrollEnable"
+                :loading="loading"
+                distance="100"
+                class="mt-9"
+                @visible="loadMoreOperations()"
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { VcRadioButton, VcButton, VcSelect, ITableColumn, VcTable, VcInfinityScrollLoader } from "@/components";
+import { useLoyalty, AddPointOperationDialog, AccountNavigation } from "@/shared/account";
+import { onMounted, ref, watch } from "vue";
+import moment from "moment";
+import { usePopup } from "@/shared/popup";
+import _ from "lodash";
+
+const { loading, userBalance, loadUserBalance, loadPointsOperations, pages, itemsPerPage, page } = useLoyalty();
+const { openPopup } = usePopup();
+
+const isInfinityScrollEnable = ref(false);
+const scrollMode = ref("pagination");
+const itemsPerPageStr = ref("10");
```

Comment:
Why this number is used as string? Please, use numbers when you need numbers and strings when you need strings to be more predictable to other developers.